cmd/apply.go

```diff
@@ -3,14 +3,17 @@ package cmd
 import (
 	"flag"
 	"fmt"
+	"os"
 
 	"github.com/driusan/dgit/git"
 )
 
 func Apply(c *git.Client, args []string) error {
 	flags := flag.NewFlagSet("apply", flag.ExitOnError)
+	flags.SetOutput(os.Stdout)
```

Comment:
Why do some of these use `os.Stdout` and other ones use `flag.CommandLine.Output()`?

Reply:
This was left over from a previous commit. Thanks, I have fixed those.